### ISSUE TITLE
kickout a SIP call

### ISSUE THREAD

**Issue by liu1083947028** [jitsi/jigasi#37]
When I kickout a SIP call before the SIP client connect in jitsi-meet conference,  the SIP client  nothing happened and it is still  ringing.

How can i do it?

**Comment by damencho** [comment]
Yep, maybe there is an issue with that. Jigasi haven't been a priority lately. So any contribution there is welcome.

**Comment by liu1083947028** [comment]
@damencho

**Comment by damencho** [comment]
Where is that code?

**Comment by liu1083947028** [comment]
@damencho 

https://github.com/jitsi/jigasi/blob/master/src/main/java/org/jitsi/jigasi/JvbConference.java

line 458

**Comment by damencho** [comment]
I don't see how this helps, this is the code to hangup the sip call. If its throwing an exception in your use case, the exception needs to be fixed.

**Comment by liu1083947028** [comment]
@damencho OK, Thanks your replay.

**Comment by liu1083947028** [comment]
@damencho 
The jigasi use 90% cpu when i make 35 calls to one jitsi-meet room. 

8 cpus  Intel(R) Xeon(R) CPU           E5620  @ 2.40GHz

Why the jigasi costs so many cpus? 

Is there any config wrong?

Looking forward to your reply.

**Comment by damencho** [comment]
Could you please use the mailinglist for questions.

In that case, every sip peer receives 34 audio streams, all of them are decoded and then mixed and encoded, and multilied by 34, this means a lot of decodes, a lot of mixing and a lot of encoding. So I would say it is normal.
